src/components/Cards/Login.vue

```diff
@@ -78,7 +78,10 @@ export default {
           console.log(err.response)
 
           // If the api gave use details of the error, then use them
-          if (err.response.data.errors) {
+          if (err.response !== undefined &&
+                err.response.data !== undefined &&
+                err.response.data.errors !== undefined
+          ) {
```

Comment:
Does someone know a shorthand for achieving the same thing? I'm sure this `if` statement can be simplified somehow

Reply:
The only thing I can think of (which would also shield this from `null`s which currently still would trip up the condition) is checking for truthiness instead:

```js
if (err.response && err.response.data && err.response.data.errors) {
```
It doesn't look like we've got Lodash or similar floating around in which case you could `_.get(err, 'response.data.errors')`.

Another option would be replacing the if/else with a try/catch that starts with
```js
const { errors } = err.response.data
```

The first option or exactly what you already have written might be our best bet.

Reply:
I like the first option, thanks!

Reply:
I was just cleaning some tabs and will leave this here for future-me: I think optional chaining could be used in such cases https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining